tests/generic_views/forms.py

```diff
@@ -15,3 +15,14 @@ class Meta:
 class ContactForm(forms.Form):
     name = forms.CharField()
     message = forms.CharField(widget=forms.Textarea)
+
+
+class ConfirmDeleteForm(forms.Form):
+    confirm = forms.BooleanField()
+
+    def clean(self):
+        cleaned_data = super(ConfirmDeleteForm, self).clean()
```

Comment:
this can be super().clean() only as python 3 don't need this explicitness

Reply:
Done.